docs/user/parser.rst

```diff
@@ -3,6 +3,35 @@ Parsing
 The first stage of Fonduer_'s pipeline is to parse an input corpus of documents
 into the Fonduer_ data model.
 
+Fonduer supports different file formats: CSV/TSV, TXT, HTML, and hOCR.
+The diagram below illustrates how files in each format are preprocessed and consumed by :class:`Parser`.
+Nodes in dark blue represent original source files.
+You have to convert some of them into the formats that Fonduer can consume:
+a scanned document (incl. non-searchable PDF) is OCRed and exported in hOCR,
+a (born-digital) PDF is converted into hOCR using tools like pdftotree_.
```

Comment:
I'd suggest "native-digital" rather than "born-digital". But fine either way.

Reply:
Thanks for your suggestion.
I'd like to keep it just because "born-digital" has its own wikipedia page (https://en.wikipedia.org/wiki/Born-digital).

Reply:
Ha, well I stand corrected. I should start using born-digital!

Reply:
It is not wrong. They are are all synonyms.
In fact, ABBYY calls it "true" PDF (https://pdf.abbyy.com/pdf-types/).